terraform/stacks/tooling/outputs.tf

```diff
@@ -317,23 +316,6 @@ output "master_bosh_secret_access_key_curr" {
   value = "${module.master_bosh_user.secret_access_key_curr}"
 }
 
-/* tooling bosh user */
-output "tooling_bosh_username" {
-  value = "${module.tooling_bosh_user.username}"
-}
-output "tooling_bosh_access_key_id_prev" {
-  value = "${module.tooling_bosh_user.access_key_id_prev}"
-}
-output "tooling_bosh_secret_access_key_prev" {
-  value = "${module.tooling_bosh_user.secret_access_key_prev}"
-}
-output "tooling_bosh_access_key_id_curr" {
-  value = "${module.tooling_bosh_user.access_key_id_curr}"
-}
-output "tooling_bosh_secret_access_key_curr" {
-  value = "${module.tooling_bosh_user.secret_access_key_curr}"
-}
-
```

Comment:
@jmcarp does it make sense to just create a new user for `semver`? one for master and tooling bosh? since it seems like we just need something to read/write to that particular bucket. wdyt?